[it] Localizzazione termine Kubernetes (#1833)
cncf/glossary: content/it/kubernetes.md

---
title: Kubernetes
status: Completed
category: tecnologia

> tags: ["infrastruttura", "fundamental", ""]

Reviewer: Non dovremmo tradurre anche questo tag?
```suggestion
tags: ["infrastruttura", "fondamenti", ""]
```

---

## Che cos'è

Kubernetes, spesso abbreviato in K8s, è un orchestratore di container open source.
Automatizza il ciclo di vita delle applicazioni containerizzate su infrastrutture moderne, fungendo da "sistema operativo per datacenter" che gestisce applicazioni su un [sistema distribuito](/it/distributed-systems/).

> Kubernetes programma i [container](/it/container/) sui [nodi](/it/nodes/) all'interno di un [cluster](/it/cluster/), raggruppando diverse risorse di infrastruttura come i load balancer, archiviazione persistente, ecc. per eseguire applicazioni containerizzate.

Reviewer: ```suggestion
Kubernetes esegue i [container](/it/container/) sui [nodi](/it/nodes/) all'interno di un [cluster](/it/cluster/), raggruppando diverse risorse di infrastruttura come i load balancer, archiviazione persistente, ecc. per eseguire applicazioni containerizzate.
```

Reviewer: mi pare che "programma" dia più un'idea di "coding"... che dite?

Kubernetes abilita automazione ed estensibilità, permettendo agli utenti di rilasciare applicazioni in modo dichiarativo (vedi sotto) in maniera riproducibile.
Le capacità di Kubernetes possono essere estese attraverso le sue [API](/it/application-programming-interface/), permettendo agli utilizzatori esperti di Kubernetes di sfruttare le sue capacità di automazione secondo le loro necessità.

## Quali problematiche affronta

> L'automazione dell'infrastruttura e la gestione delle configurazioni in modo dichiarativo sono concetti importanti da diverso tempo, ma sono diventati sempre più importanti da quando il [cloud computing](/it/cloud-computing/) ha cominciato a diffodersi.

Reviewer: ```suggestion
L'automazione dell'infrastruttura e la gestione delle configurazioni in modo dichiarativo sono concetti importanti da diverso tempo, ma sono diventati sempre più importanti da quando il [cloud computing](/it/cloud-computing/) ha cominciato a diffondersi.
```

> Man mano che la richiesta di risorse computazionali aumenta e le organizzazioni hanno bisogno di fornire più capacità operative con meno ingegneri, servono nuove tecnologie e nuovi metodi di lavoro.

Reviewer: meno ingegneri è una parte esecrabile, comunque  :D

## In che modo aiuta

Analogamente agli strumenti tradizionali di [infrastructure as code](/it/infrastructure-as-code/), Kubernetes aiuta con l'automazione e ha il vantaggio di lavorare con i container.

> I container sono più resistenti alla deriva della configurazione rispetto a macchine [virtuali](/it/virtual-machine/) o fisiche.

Reviewer: in genere i "drift" sono delle differenze. Qui probabilmente occorre porre l'accento sul fatto che non sono volute. io direi...
```suggestion
I container sono meno inclini ai disallineamenti di configurazione rispetto a macchine [virtuali](/it/virtual-machine/) o fisiche.
```

Reviewer 2: @ugho16 qua se accetti le modifiche di Simone, possiamo andare avanti, credo
